requirements.in

```diff
@@ -28,4 +28,4 @@ nltk
 gnews
 trafilatura
 python-facebook-api
-
+pre-commit
```

Comment:
We can setup dev requirement in other PR to handle all development related dependencies like pytest, black, mypy etc

Reply:
Okay, I will first send a PR to add development requirements and then I will finish this PR.